deployment/GS01SUMO_Deploy.csv

```diff
@@ -85,73 +85,73 @@ NBP-15-11,,NBP16-10,,GS01SUMO-RII11-02-PCO2WC052,2,1,2015-12-14T20:20:00,2016-12
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD11-03-HYDGN0000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-HYDGN0-41867,-54.40408,-89.20691,,0,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD11-04-DOSTAD000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-DOSTAD-00495,-54.40408,-89.20691,,1,4588,
-NBP-15-11,,NBP16-10,,GS01SUMO-SBD11-05-SPKIRB000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-SPKIRB-00300,-54.40408,-89.20691,,-5,4588,
-NBP-15-11,,NBP16-10,,GS01SUMO-SBD11-06-METBKA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-METLGR-00022,-54.40408,-89.20691,,-5,4588,
+NBP-15-11,,NBP16-10,,GS01SUMO-SBD11-05-SPKIRB000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-SPKIRB-00300,-54.40408,-89.20691,,-6,4588,
+NBP-15-11,,NBP16-10,,GS01SUMO-SBD11-06-METBKA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-METLGR-00022,-54.40408,-89.20691,,0,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD11-08-NUTNRB000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-NUTNRB-00277,-54.40408,-89.20691,,1,4588,
-NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-00-DCLENG000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,OL000078,-54.40408,-89.20691,,0,4588,
+NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-00-DCLENG000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGCON-BDCL12-50003,-54.40408,-89.20691,,0,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-01-OPTAAD000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-OPTAAD-00254,-54.40408,-89.20691,,1,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-02-FLORTD000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-FLORTD-01319,-54.40408,-89.20691,,1,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-03-HYDGN0000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-HYDGN0-42356,-54.40408,-89.20691,,0,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-04-PCO2AA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-PCO2AA-31124,-54.40408,-89.20691,,0,4588,
 NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-05-WAVSSA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-WAVSSA-04571,-54.40408,-89.20691,,0,4588,
-NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-06-METBKA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-METLGR-00026,-54.40408,-89.20691,,-5,4588,
-NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-08-FDCHPA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-FDCHPA-44901,-54.40408,-89.20691,,-5,4588,
+NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-06-METBKA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-METLGR-00026,-54.40408,-89.20691,,0,4588,
+NBP-15-11,,NBP16-10,,GS01SUMO-SBD12-08-FDCHPA000,2,1,2015-12-14T20:20:00,2016-12-12T07:58:00,CGMGS-01SUMO-00002,,CGINS-FDCHPA-44901,-54.40408,-89.20691,,-6,4588,

```

Comment:
Deployment 2 ADCP depth should be 500 (not 501).

Reply:
fixed

Reply:
Looks good.

Reply:
Deployment 2 looks good!